processor/intracesampler/README.md

```diff
@@ -0,0 +1,70 @@
+# In-Trace Sampling Processor
+
```

Comment:
Doesn't need to be in this pr, but the status table can now be auto-generated, you can look at the haproxy files in [this change](https://github.com/open-telemetry/opentelemetry-collector-contrib/commit/7ab7c53d32806e3986f353eee7e659349ac693a6) to see how it's done

Reply:
mind if we leave this for the next PR? I'll take care of it.

Reply:
Alright, I think you said yes. Closing this thread.